server/task-cassandra/src/main/scala/org/apache/james/task/eventsourcing/cassandra/TasksSerialization.scala

```diff
@@ -0,0 +1,150 @@
+/** **************************************************************
+  * Licensed to the Apache Software Foundation (ASF) under one   *
+  * or more contributor license agreements.  See the NOTICE file *
+  * distributed with this work for additional information        *
+  * regarding copyright ownership.  The ASF licenses this file   *
+  * to you under the Apache License, Version 2.0 (the            *
+  * "License"); you may not use this file except in compliance   *
+  * with the License.  You may obtain a copy of the License at   *
+  *                                                              *
+  * http://www.apache.org/licenses/LICENSE-2.0                   *
+  *                                                              *
+  * Unless required by applicable law or agreed to in writing,   *
+  * software distributed under the License is distributed on an  *
+  * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+  * KIND, either express or implied.  See the License for the    *
+  * specific language governing permissions and limitations      *
+  * under the License.                                           *
+  * ***************************************************************/
+
+package org.apache.james.task.eventsourcing.cassandra
+
+import scala.collection.JavaConverters._
+import org.apache.james.eventsourcing.eventstore.cassandra.JsonEventSerializer
+import org.apache.james.eventsourcing.eventstore.cassandra.dto.{EventDTO, EventDTOModule}
+import org.apache.james.eventsourcing.{Event, EventId}
+import org.apache.james.server.task.json.JsonTaskSerializer
+import org.apache.james.server.task.json.dto.TaskDTO
+import org.apache.james.task.eventsourcing._
+import org.apache.james.task.{Task, TaskId}
+import play.api.libs.json._
```

Comment:
could you explain how play json is articulated with JsonSerializer ?

Reply:
Oh, the commits are messed-up in github at least, I think I know understand the design